inmanta.spec

```diff
@@ -0,0 +1,83 @@
+# Use release 0 for prerelease version.
+%define release 0
```

Comment:
```suggestion
%define release 1
```

Reply:
This is on master, so a prelease. That have to be 0

Reply:
Note that the Makefile set this value to 0 when a non-stable release is requested. So it should be set to 1 by default and it's overridden for non-stable releases. I agree this is weird.

Reply:
ah, that is fine. But we might need to add a comment in the specfile :-)

Reply:
damnit, I got this email but then afterwards I did not see the comment in the PR. Fix now (I hope)